### PR TITLE
Support iOS 14 Xcode 12

### PR DESCRIPTION
**Pull request by IlyaBausovAkvelon**
1. Removed 'enableAccessibility' from bridge.
2. Added new devices and os versions.

### CODE REVIEW

File: FBControlCore/Diagnostics/FBBatchLogSearch.m [hunk 1]
```diff
@@ -197,8 +197,8 @@ - (id)jsonSerializableRepresentation
   for (NSArray *key in self.mapping) {
     mappingDictionary[key] = [self.mapping[key] valueForKey:@"jsonSerializableRepresentation"];
   }
-  BOOL lines = self.options & FBBatchLogSearchOptionsFullLines;
-  BOOL first = self.options & FBBatchLogSearchOptionsFirstMatch;
+  BOOL lines = (self.options & FBBatchLogSearchOptionsFullLines) ? YES : NO;
```
Comment (ivan-nosar) [inline FBControlCore/Diagnostics/FBBatchLogSearch.m:200]: Why do we need this change here? Is it an automatically generated code?

File: FBControlCore/Diagnostics/FBBatchLogSearch.m [hunk 2]
```diff
@@ -299,8 +299,8 @@ + (NSDictionary *)searchDiagnostics:(NSArray<FBDiagnostic *> *)diagnostics withP
 
 + (NSArray<NSString *> *)search:(FBDiagnosticLogSearch *)search withOptions:(FBBatchLogSearchOptions)options
 {
-  BOOL lines = options & FBBatchLogSearchOptionsFullLines;
-  BOOL first = options & FBBatchLogSearchOptionsFirstMatch;
+  BOOL lines = (options & FBBatchLogSearchOptionsFullLines) ? YES : NO;
```
Comment (ivan-nosar) [inline FBControlCore/Diagnostics/FBBatchLogSearch.m:302]: The same as the comment above

File: FBSimulatorControl/Management/FBSimulatorBridge.m [hunk 3]
```diff
@@ -55,7 +55,6 @@ + (nullable instancetype)bridgeForSimulator:(FBSimulator *)simulator error:(NSEr
   }
 
   // Load Accessibility, return early if this fails
-  [bridge enableAccessibility];
```
Comment (ivan-nosar) [inline FBSimulatorControl/Management/FBSimulatorBridge.m:58]: Do we **really** need this line to be deleted? In my understanding, if we decided to remove some method call due to its deprecation - we have to implement its replacement that will do the same things as the original method

**Review by ivan-nosar — APPROVED** [review]
@IlyaBausovAkvelon could you please reply to all the comments above?

File: FBControlCore/Diagnostics/FBBatchLogSearch.m (hunk 1 above)
Comment (IlyaBausovAkvelon) [reply FBControlCore/Diagnostics/FBBatchLogSearch.m:200]: It was needed to resolve warnings about deprecations.

File: FBControlCore/Diagnostics/FBBatchLogSearch.m (hunk 2 above)
Comment (IlyaBausovAkvelon) [reply FBControlCore/Diagnostics/FBBatchLogSearch.m:302]: It was needed to resolve warnings about deprecations.

File: FBSimulatorControl/Management/FBSimulatorBridge.m (hunk 3 above)
Comment (IlyaBausovAkvelon) [reply FBSimulatorControl/Management/FBSimulatorBridge.m:58]: Fixed this. Added checking that bridge has this selector before invoking it.

**Comment by IlyaBausovAkvelon** [comment]
@ivan-nosar answered and resolved.

**Review by AndreyMaslennikov — COMMENTED** [review]
LGTM

**Pull request merged**